### PR TITLE
[HOLD] Pre-commit Fizz Validation

### PR DESCRIPTION
**Pull request by pjdufour-truss**
## Description

Requires: https://github.com/gobuffalo/fizz/pull/54

This PR adds pre-commit validation for our fizz files.  Requires an update to the fizz git repo.

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

```shell
make bin/fizz-validate
pre-commit run -a fizz-validate
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/tree/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/tree/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/tree/master/docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
  * [ ] Secure migrations have been tested using `scripts/run-prod-migrations`
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

None

## Screenshots

None

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -179,6 +179,9 @@ bin/callgraph: .check_go_version.stamp .check_gopath.stamp
 bin/chamber: .check_go_version.stamp .check_gopath.stamp
 	go build -ldflags "$(LDFLAGS)" -o bin/chamber github.com/segmentio/chamber
 
+bin/fizz-validate: .check_go_version.stamp .check_gopath.stamp
+	go build -ldflags "$(LDFLAGS)" -o bin/fizz-validate github.com/transcom/mymove/cmd/fizz-validate
```
Comment (chrisgilmerproj) [inline Makefile]: Let's move this down to the section on `MilMove Targets` which is where the other binaries from this project are.

**Comment by pjdufour-truss** [comment]
Moved make target and pinged about merging PR to fizz.

**Comment by pjdufour-truss** [comment]
We're going to ice this PR for a month (until July 19) to check for a fix to fizz.

**Comment by chrisgilmerproj** [comment]
Closing until the issue with Fizz is resolved. Infra has a ticket to reopen this if it does.

**Pull request closed without merging**